index.js

```diff
@@ -52,6 +52,15 @@ function getOptions(addonContext) {
     }
   }
 
+  // add babel-plugins from the registry to the options.plugins
+  var pluginWrappers = addonContext.registry.load('babel-plugin');
```

Comment:
I would prefer that we stash the registry instance we are passed for `parent` in the `setupPreprocessorRegistry` hook.

Something like:

```javascript
setupPreprocessorRegistry: function(type, registry) {
  if (type === 'parent') {
    this.parentRegistry = registry;
  }
}
```

Then you should be able to call `addonContext.parentRegistry.load` without issue (or hacks).